spec/messenger_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'messenger'
+
+describe Messenger do
+  subject(:messenger) { described_class.new }
+
+  describe '#send_message' do
+    it 'can send a message' do
+      allow(messenger).to receive(:send_message).and_return("Success")
+      message = "Your Dominos pizza is on its way!"
+      expect(messenger.send_message(message)).to eq "Success"
```

Comment:
Agreed, but the stub was only put in place to prevent Rspec from sending text messages (as mentioned in requirements) after having checked that the test passes. I cannot delete the test because I may need to comment the stubbing and test real functionality. I didn't want to give read access to any of the ENV variables hence couldn't test initialize. But after implementing the changes you suggested to SMS class, I can.